app/jenkins-for-jira-ui/src/components/ManageConnection/ManageConnection.tsx

```diff
@@ -71,7 +71,7 @@ const ManageConnection = () => {
 		try {
 			const { name, secret: retrievedSecret } = await getJenkinsServerWithSecret(uuid);
 			setServerName(name);
-			setSecret(retrievedSecret!);
+			setSecret(retrievedSecret);
```

Comment:
Another warning fix - was giving a non-null assertion warning (trying to clean up bit by bit whenever I can).